docs/workspaces.md

```diff
@@ -534,8 +534,8 @@ workspaces:
 
 ##### `csi`
 
-This is an alpha feature. The `enable-api-fields` feature flag [must be set to `"alpha"`](./install.md)
-for csi volume source to function.
+This is a beta feature in v1. The `enable-api-fields` feature flag [must be set to `"beta"`](./install.md)
+for csi volume source to function in v1.
```

Comment:
could it be confusing to have this in docs when we haven't released v1 yet?

Reply:
Makes sense, thanks @jerop 
I will delete this for now. 
Also noted this in https://github.com/tektoncd/pipeline/issues/5663